### ISSUE TITLE
S3 node output is following a faulty format

### ISSUE THREAD

**Issue by arladmin** [n8n-io/n8n#5502]
**Describe the bug**

Data is not shown/accessible in the output of the S3 node.

![image](https://user-images.githubusercontent.com/64059154/219582586-20daeba8-46ae-4062-966b-a3fdb7ed9969.png)

![image](https://user-images.githubusercontent.com/64059154/219582299-04749d82-3a1b-44aa-a03a-b34ef82f5c89.png)

![image](https://user-images.githubusercontent.com/64059154/219582474-9e6feef6-45d1-40fc-b72f-caee9fd548bd.png)

**Expected behavior**
The items in the output should have been visible as well as accessible by further nodes.

Upon inspecting in the code node, and by printing the result of `$input.all()` in the console, the following data structure is found:
![image](https://user-images.githubusercontent.com/64059154/219583493-9cae7886-10a4-4aa3-bbee-c578a375b013.png)

Basically, the output is malformed as it is missing the default `json` property and the S3 node's output data has been pushed to the top level.

**Environment (please complete the following information):**

- OS: [e.g. Ubuntu Linux 22.04] Windows 10
- n8n Version [e.g. 0.200.1] 0.216.0
- Node.js Version [e.g. 16.17.0] default
- Database system [e.g. SQLite; n8n uses SQLite as default otherwise changed] Postgres
- Operation mode [e.g. own; operation modes are `own`, `main` and `queue`. Default is `own`] main

**Additional context**

**Comment by Joffcom** [comment]
Hey @arladmin,

Is it just Bucket > Get Many failing for you?

We have an internal ticket opened for this already which we are tracking as `NODE-137`

**Comment by arladmin** [comment]
Hi @Joffcom,

Yes, currently that's the only scenario that's failing for me.

**Comment by Joffcom** [comment]
Hey @arladmin,

Perfect, I have popped in a fix for that and assuming it is reviewed today / tomorrow it will be available in the next release.

**Comment by arladmin** [comment]
> Hey @arladmin,
> 
> Perfect, I have popped in a fix for that and assuming it is reviewed today / tomorrow it will be available in the next release.

Awesome, thanks!

**Comment by Joffcom** [comment]
Hey @arladmin,

This has been merged now and will be available in the next release, For now I am going to mark this one as closed. When the release happens one of us will drop a follow up message on here to let you know.

**Comment by janober** [comment]
Fix got released with `n8n@0.217.1`
